### PR TITLE
dark-mode : right click Context Menu : )

### PR DESCRIPTION
**Pull request by yashash1511**
image

### CODE REVIEW

**Comment by yashash1511** [comment]
### DO NOT MERGE (WIP)

**Comment by blueedgetechno** [comment]
can we not change the icons color to gray, instead of black so we will not have to deal with inverting its colors

**Comment by Posandu** [comment]
The border should be changed @yashash1511

**Comment by Posandu** [comment]
> 
> # DO NOT MERGE (WIP)

My culture is being developed.. lol

**Comment by yashash1511** [comment]
> can we not change the icons color to gray, instead of black so we will not have to deal with inverting its colors

no I have better way will push soon it retains colors :)

**Comment by yashash1511** [comment]
> can we not change the icons color to gray, instead of black so we will not have to deal with inverting its colors

image

but this looks like real

**Comment by yashash1511** [comment]
> The border should be changed @yashash1511

okey will do :)

**Comment by yashash1511** [comment]
> can we not change the icons color to gray, instead of black so we will not have to deal with inverting its colors

image

but this looks more like real :)

**Comment by blueedgetechno** [comment]
but still you are brute forcing the css to not include 11 and 12th children, i.e terminal and about Icon.

If I add another command in the context menu, lets say open with vscode then I have to manually edit the css which is not the right way towards making it dynamic.

File: src/components/menu/menu.scss [hunk 1]
```diff
@@ -21,16 +24,29 @@
...
+body[data-theme="dark"] .actmenu {
+  --bg1: rgba(44, 44, 44, 0.8);
+  --bg2: rgba(255, 255, 255, 0.08);
+
+  .menuopt img {
+    filter: hue-rotate(180deg) invert(1);
+  }
+  .menuopt:nth-child(11) img,
+  .menuopt:nth-child(12) img {
+    filter: none;
```
Comment (Posandu) [inline src/components/menu/menu.scss:36]: ?

File: src/components/menu/menu.scss [hunk 2]
```diff
@@ -21,16 +24,29 @@
   }
 }
 
+body[data-theme="dark"] .actmenu {
+  --bg1: rgba(44, 44, 44, 0.8);
+  --bg2: rgba(255, 255, 255, 0.08);
+
+  .menuopt img {
+    filter: hue-rotate(180deg) invert(1);
```
Comment (Posandu) [inline src/components/menu/menu.scss:32]: This is useless

File: src/components/menu/menu.scss (hunk 1 above)
Comment (blueedgetechno) [reply src/components/menu/menu.scss:36]: these are to prevent terminal icon and about icon from hue-rotate and invert effect

Comment (blueedgetechno) [reply src/components/menu/menu.scss:36]: which can temporary work but will be issue to resolve later

File: src/components/menu/menu.scss (hunk 2 above)
Comment (blueedgetechno) [reply src/components/menu/menu.scss:32]: Useles will be a harsh word, I would say it is replacable to something better.

**Comment by yashash1511** [comment]
@blueedgetechno

|image |  image |
|---|---|
| dark | light |

this is how it will look if grayed out is it fine ?

**Comment by blueedgetechno** [comment]
It looks good, you can push for its hosted version

**Comment by yashash1511** [comment]
| image | image |
|---|---|

**Comment by blueedgetechno** [comment]
I mean keep the blue part blue, like this

image

but dont worry about, I will do it. You can simply work with grayscale. But by editing the icon image gray scale manually and not by using filter

**Comment by yashash1511** [comment]
how do you keep the blue part

**Comment by blueedgetechno** [comment]
I edited it in photoshop

**Comment by yashash1511** [comment]
> I edited it in photoshop

ohh can you pls do this part I dont hav photoshop

**Comment by blueedgetechno** [comment]
sure sure, you simple delete these lines and we are good to go

```css
menuopt img {
    filter: hue-rotate(180deg) invert(1);
}

.menuopt:nth-child(11) img,
.menuopt:nth-child(12) img {
    filter: none;
}
```

**Comment by yashash1511** [comment]
> sure sure, you simple delete these lines and we are good to go
> 
> ```css
> menuopt img {
>     filter: hue-rotate(180deg) invert(1);
> }
> 
> .menuopt:nth-child(11) img,
> .menuopt:nth-child(12) img {
>     filter: none;
> }
> ```

okey will do :)

**Comment by yashash1511** [comment]
removed border :)

**Pull request merged**